src/app/meds-n-labels/activity-2/activity-2.component.ts

```diff
@@ -138,19 +150,20 @@ export class MemoryGameComponent implements OnInit {
             const med = medName.substr(0, medName.lastIndexOf('.'));
             for (let i = 0; i < DIAGNOSIS.length; i++) {
                 if (DIAGNOSIS[i].name === med) {
-                    this._sharedData.customAlert('Congratulations!<br>You matched a pair', DIAGNOSIS[i].desc, 'success');
+                    this._sharedData.customAlert(this.language.alerts.title, DIAGNOSIS[i].desc, 'success');
                     this.isMatchArr[this._secondchoice] = false;
                     this.isMatchArr[this._firstchoice] = false;
                     break;
                 }
             }
             this._matches++;
             if (this._matches === 8) {
-                this._sharedData.customAlert('Good job!', 'You completed this activity!', 'success');
+                this._sharedData.customSuccessAlert(this.alerts.activitySuccessMsg, this.alerts.activitySuccessTitle);
                 this._status = {stage: 3, activity: 2};
                 this._dashboardService.updateProgressStatus(this._status).subscribe(response => {});
                 this.activityComplete = true;
             }
+            return;
```

Comment:
Yes, I had removed it in my previous PR, but then I realised that it doesn't work well without it, it is to break out of the function execution when a certain condition satisfies.